label/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds some report template and report actions associated with them.
```

Comment:
Please be more specific and detail the unique report the module is adding. Include the specifications of the label sheet it produces.